package-lock.json

```diff
@@ -4712,11 +4712,11 @@
       }
     },
     "node_modules/@probot/pino": {
-      "version": "2.5.0",
-      "resolved": "https://registry.npmjs.org/@probot/pino/-/pino-2.5.0.tgz",
-      "integrity": "sha512-I7zI6MWP1wz9qvTY8U3wOWeRXY2NiuTDqf91v/LQl9oiffUHl+Z1YelRvNcvHbaUo/GK7E1mJr+Sw4dHuSGxpg==",
+      "version": "2.3.5",
```

Comment:
This reverts the changes from dependabot in #35 please revert back the lock file.

Thanks.